### DISCUSSION TITLE
How to continue a training from a LoRA checkpoint ?

### DISCUSSION THREAD

**Discussion by l-k-11235** [eole-nlp/eole#62] [Q&A]
I tried to relaunch the finetuning with the same config but it restarted the training from scratch. Moreover, as the checkpoint directory only  contains the lora weights, it may require some adaptations.

**Comment by vince62s** [comment]
You cannot do this. You will need to merge your lora weight first and then train_from the merged model.

**Reply by l-k-11235** [reply]
I have tried but it does not perform well (see  #64 ).
Moreover it is quite different, as the LoRA matrixes and the optimizer states will be reset.

**Reply by vince62s** [reply]
There is no other path at the moment and doubtful about the other one to be implemented.
